wolfcrypt/src/ed448.c

```diff
@@ -541,11 +616,21 @@ int wc_ed448_init(ed448_key* key)
         XMEMSET(key, 0, sizeof(ed448_key));
 
         fe448_init();
+
+        key->heap = heap;
     }
 
     return ret;
 }
 
+/* Initialize the ed448 private/public key.
+ *
+ * key  [in]  Ed448 key.
+ * returns BAD_FUNC_ARG when key is NULL
+ */
+int wc_ed448_init(ed448_key* key) {
+    return wc_ed448_init_ex(key, NULL);
```

Comment:
If you are adding _ex please include `int devId` for future. Defaults to `INVALID_DEVID`.